### DISCUSSION TITLE
remove 'at' in Refresh tokens 'created at'

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#9137] [Configuration] [answered]
a tiny thing, should be an easy fix ;-)

`Created at 13 weeks` ago should be `created 13 weeks ago` ? Same applies for the seconds line.

<img width="587" alt="Schermafbeelding 2021-05-08 om 16 19 35" src="https://user-images.githubusercontent.com/33354141/117542589-84959a80-b019-11eb-8d47-231687c23ace.png">

**Comment by spacegaier** [comment, marked as answer]
PR created: https://github.com/home-assistant/frontend/pull/9138
